### DISCUSSION TITLE
Web UI access via beryl

### DISCUSSION THREAD

**Discussion by loggerhead-turtle** [SmoothWAN/SmoothWAN#38] [General & Help]
Hopefully this makes sense, I setup my pi (bonding is working, too with speedify) and then connected my network cable between the pi and my beryl router. Devices connected to the router get internet access, but I can't log into the UI of the raspberry pi to adjust the speedify usb priority. I have to log into the UI using the 172.17.17.2 IP address to make changes.

What am I missing?

Thanks!

**Comment by TalalMash** [comment]
Hi!
Do you mean you can't login to the Pi UI via the Beryl, but you can do so when connected directly to Pi?

**Reply by loggerhead-turtle** [reply]
That is correct. Desktop behind beryl gets internet. But, doesn't have access to the pi UI. Same desktop, if I connect to the smwartwansetup wifi, will connect to the pi UI

**Reply by TalalMash** [reply]
Seems like NAT is enabled on the WAN side, if you can't disable it, simply follow https://github.com/TalalMash/SmoothWAN/discussions/18
to setup bridged AP, it's the proper way, I'll update the README next release.
